packages/meditrak-server/src/routes/importSurveyResponses/assertCanImportSurveyResponses.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import flattenDeep from 'lodash.flattendeep';
+
+export const assertCanImportSurveyResponses = async (
+  accessPolicy,
+  models,
+  entitiesByPermissionGroup,
+) => {
+  const allEntityCodes = flattenDeep(Object.values(entitiesByPermissionGroup));
+  const allEntities = await models.entity.findManyByColumn('code', allEntityCodes);
```

Comment:
```suggestion
  const allEntities = await models.entity.find({ code: allEntityCodes });
```
I like your thinking here, but this will do the same, is more concise and creates fewer variations of how to do the same thing with the db

Reply:
I just realised you may have done this to utilise the functionality that avoids overloading the number of bindings in a single db query - was that why?

Reply:
ah yep, coz I thought that we there can be a huge number of entities when we import them (eg Laos Schools)... 😬

Reply:
nice, sorry I missed that at the time! so it's just the slack discussion required :-)